.gitignore

```diff
@@ -2,3 +2,4 @@
 *.iml
 target/*
 logs/*
+lib/*
```

Comment:
those files are still in the PR though
I can't understand why do you need them and what do they mean?
can we remove them?

Reply:
ok, I'll remove it. I don't know how it's happen🙃